### ISSUE TITLE
Need tracing e2e test

### ISSUE THREAD

**Issue by douglas-reid** [istio/istio#3947]
Istio currently doesn't have anything that verifies tracing data in Zipkin that is generated per request (including ingress and Mixer spans). One should be added.

**Comment by smawson** [comment]
Doug OK for me to try taking this for the fixit week?

**Comment by douglas-reid** [comment]
@smawson absolutely. I was going to start on it if I was blocked on current paths later this week, but am happy to have someone else own this. Enjoy!

**Comment by smawson** [comment]
@douglas-reid There is an existing zipkin_test.go under the pilot e2e tests that seems to verify zipkin tracing, is more needed beyond what that is doing?

**Comment by douglas-reid** [comment]
@smawson I think we want one that uses a more complex app deployment than the pilot e2e setup for a few reasons:
- Validating Mixer tracing
- User reports of often only getting one span per trace
- Future work on configuring trace sampling (ability to turn off tracing for particular destinations, setting tracing to 1%).

I don't believe the `zipkin_test.go` will cover all of that.
